Framework/CurveFitting/src/Algorithms/ConvolutionFit.cpp

```diff
@@ -255,7 +255,7 @@ ConvolutionFit<QENSFitSequential>::seeAlso() const {
 template <>
 const std::vector<std::string>
 ConvolutionFit<QENSFitSimultaneous>::seeAlso() const {
-  return {"QENSSimultaneousFit"};
+  return {"QENSFitSimultaneous"};
```

Comment:
@brandonhewer This changes looks like the correct thing to do based on the class name but are there any implications for the MSG GUIs?

Reply:
It's only the seeAlso string, I don't think they are used anywhere but in the documentation.

Reply:
Good point. Thanks @NickDraper.